packages/composites/dydx-rewards/README.md

```diff
@@ -11,7 +11,8 @@ The adapter takes the following environment variables:
 | Required? |             Name              |                                                              Description                                                              | Options |                 Defaults to                  |
 | :-------: | :---------------------------: | :-----------------------------------------------------------------------------------------------------------------------------------: | :-----: | :------------------------------------------: |
 |    ✅     |         `PRIVATE_KEY`         |                                         Private key of account used to make special callbacks                                         |         |                                              |
-|    ✅     |      `ETHEREUM_RPC_URL`       |                                                     Ethereum Mainnet RPC URL node                                                     |         |                                              |
+|           |      `ETHEREUM_RPC_URL`       |                                                         Ethereum RPC URL node                                                         |         |                                              |
```

Comment:
Shouldn't the ETHEREUM_RPC_URL still be required here? (missing ✅)

Reply:
Oh, this one is mistake, thanks. Will fix